neard/src/runtime.rs

```diff
@@ -529,6 +538,30 @@ impl NightshadeRuntime {
 
         Ok(result)
     }
+
+    #[allow(unused_variables)]
+    fn check_duplicate_transaction(
```

Comment:
The more canonical name will `is_duplicate_transaction`
```suggestion
    fn is_duplicate_transaction(
```